android/301Project/app/src/main/java/com/example/a301project/IngredientActivity.java

```diff
@@ -201,34 +230,34 @@ public void sortDataBySpinner() {
         );
         ingredientAdapter.notifyDataSetChanged();
     }
-
+    /**
+     * Method for adding ingredients
+     * trigger when Add button clicked
+<<<<<<< HEAD
+     * @param ingredient {@link Ingredient} an ingredient to be added
+=======
+     * @param ingredient {@link Ingredient} an ingredient
+>>>>>>> 9af36dcb74ba57f887d7bdfc49103e27fadc927d
+     * @return void
+     */
     public void addIngredient(Ingredient ingredient) {
-        /**
-         * Method for adding ingredients
-         * trigger when Add button clicked
-         * @return void
-         */
         ingredientAdapter.add(ingredient);
         ingredientController.addIngredient(ingredient);
     }
 
-    public void deleteIngredient(Ingredient ingredient) {
-        /**
-         * Method for deleting ingredient
-         * attempting to implement recycleview
-         * @return void
-         */
-        ingredientAdapter.remove(ingredient);
-        ingredientController.removeIngredient(ingredient);
-    }
```

Comment:
Also was this supposed to be deleted? I don't know how Ingredient works very well

Reply:
I deleted it and it works fine, this function was never called because deleteIngredient was being used in the AddEditIngredientFragment